### PR TITLE
HPCC-14869 Add ApplicationValues to WURun

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@afishbeck Please review

File: esp/services/ws_workunits/ws_workunitsHelpers.cpp [hunk 1]
```diff
@@ -2952,6 +2952,13 @@ void WsWuHelpers::submitWsWorkunit(IEspContext& context, IConstWorkUnit* cw, con
         }
     }
 
+    ForEachItemIn(ii, *applications)
```
Comment (afishbeck) [inline esp/services/ws_workunits/ws_workunitsHelpers.cpp]: Should first check if applications is null.

**Comment by afishbeck** [comment]
@wangkx Other than adding a check for null, looks fine.

**Comment by wangkx** [comment]
@afishbeck I add the null check. Please review.

**Comment by afishbeck** [comment]
@wangkx looks good

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to merge.

**Pull request merged**